content/applications/finance/payment_acquirers/adyen.rst

```diff
@@ -2,15 +2,15 @@
 Adyen
 =====
 
-`Adyen <https://www.adyen.com/>`_ is a Dutch-based company that offers several online payment
+`Adyen <https://www.adyen.com/>`_ is a Dutch company that offers several online payment
 possibilities.
 
+.. seealso::
+   :ref:`payment_acquirers/add_new`
+
 Configuration
 =============
 
-.. seealso::
-   - :ref:`payment_acquirers/add_new`
-
```

Comment:
Configuration title still under the `seealso` ;)

Reply:
Done! :)